eoscostarica/eos-rate
# Fix/tooltips description 269

### GH Issue

#269 Add Category Tooltips Description Text 

### Steps to test

1. Log in with scatter
1. Open BP Ratings Page
1. Toggle between english and spanish and check translations

### Checklist
